nurseconnect/settings/base.py

```diff
@@ -304,3 +301,18 @@
 EMAIL_PORT = environ.get('EMAIL_PORT', 25)
 EMAIL_HOST_USER = environ.get('EMAIL_HOST_USER', '')
 EMAIL_HOST_PASSWORD = environ.get('EMAIL_HOST_PASSWORD', '')
+
+# Molo comments settings
+COMMENTS_APP = 'molo.commenting'
+COMMENTS_FLAG_THRESHHOLD = 3
+COMMENTS_HIDE_REMOVED = False
+
+# Password reset - security questions
+SECURITY_QUESTION_1 = environ.get(
+    'SECURITY_QUESTION_1', 'Name of the city you were born in')
+SECURITY_QUESTION_2 = environ.get(
+    'SECURITY_QUESTION_2', 'Name of your primary school')
+
+# Comment Filtering Regexes
+REGEX_PHONE = r'.*?(\(?\d{3})? ?[\.-]? ?\d{3} ?[\.-]? ?\d{4}.*?'
```

Comment:
Just double check this please, international numbers vary.  Not sure what standard you are working to, also exclude selecting `.` delimiters.  Also some parts of this seem unnecessary.